src/nih_wayfinding/app/index.html

```diff
@@ -101,6 +102,7 @@ <h2 class="navbar-actionheader" ng-cloak ui-sref="profiles">
     <!-- endbuild -->
 
     <!-- build:js({.tmp,app}) scripts/scripts.js -->
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/Leaflet.awesome-markers/2.0.2/leaflet.awesome-markers.js"></script>
```

Comment:
Also any cdn'ed javascript/css files should be outside of the `<!-- build:filetype destFilename -->` tags in index.html